velox/dwio/common/tests/TypeTests.cpp

```diff
@@ -142,6 +155,10 @@ TEST(TestType, selectedType) {
   EXPECT_EQ(1, cutType->childAt(0)->maxId());
   EXPECT_EQ(11, cutType->childAt(1)->id());
   EXPECT_EQ(11, cutType->childAt(1)->maxId());
+
```

Comment:
Move this to the beginning of the test, create a `typeWithIdCopy` by calling `buildSelectedType()`, and `assertEqualTypeWithId(typeWithIdCopy, typeWithId) for every test case after the selector is applied.